<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_word_cloud">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>125</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>311</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>당일 기사를 기반으로 제공
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>330</y>
      <width>350</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;코스피 TOP 종목&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;금융상품 키워드 제공&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_result">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>31</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>31</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>기업 명</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="search_space">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>357</y>
      <width>400</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="volume_Top">
     <attribute name="title">
      <string>  거래량  </string>
     </attribute>
     <widget class="QTableWidget" name="volumnTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>240</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>          종목명           </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  현재가  </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> 전일비 </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  등락률  </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>        거래량         </string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="up_Top">
     <attribute name="title">
      <string>   상승   </string>
     </attribute>
     <widget class="QTableWidget" name="upTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>240</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>          종목명           </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  현재가  </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> 전일비 </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  등락률  </string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="down_Top">
     <attribute name="title">
      <string>   하락   </string>
     </attribute>
     <widget class="QTableWidget" name="downTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>240</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>          종목명           </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  현재가  </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> 전일비 </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  등락률  </string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="total_Top">
     <attribute name="title">
      <string>시가총액 상위</string>
     </attribute>
     <widget class="QTableWidget" name="totalTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>240</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>          종목명           </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  현재가  </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> 전일비 </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  등락률  </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  시가총액(억) </string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_crawl_day">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>오늘</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_result</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>search_stock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_result</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_main_to_second()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search_stock()</slot>
  <slot>btn_main_to_second()</slot>
 </slots>
</ui>
